doc/articles/features/windows-system-profile.md

```diff
@@ -0,0 +1,38 @@
+﻿# Uno Support for Windows.System.Profile APIs
+
+## `AnalyticsInfo`
+
+### `DeviceForm` property
+
+The `DeviceForm` property now returns string from the following set:
+
+- Mobile
+- Tablet
+- Television
+- Car
+- Watch
+- VirtualReality
+- Desktop
+- Unknown
+
+In case of `WebAssembly` this property currently always returns `Unknown` as there is no reliable way to check for device form factor via JavaScript.
+
+## `VersionInfo.DeviceFamily`
+
+This string is in the form of `OSType.DeviceForm` where `OSType` is:
+
+- **Android** - `Android`
+- **iOS/iPadOS** - `Apple` 
+- **WASM** - browser name (suppored options are `Firefox`, `Edge`, `Chrome`, `Opera`, and a generic `Browser` option)
```

Comment:
Not sure about this one. On the one hand, it's useful to expose the browser vendor. On the other hand, I think the more common use case is to know 'this app is running in WebAssembly, and not some other Uno target', to which end a generic `Browser` is more useful.

Reply:
That makes sense. I can make it just `Browser` or `WebAssembly`. The browser detection can be imprecise anyway.